src/CompileTime.hx

```diff
@@ -64,6 +70,17 @@ class CompileTime
         return toExpr(obj);
     }
 
+    #if yaml
+    macro public static function parseYamlFile(path:String) {
+      var content = loadFileAsString(path);
+      var data = Yaml.parse(content, Parser.options().useObjects());
+      var s = haxe.Json.stringify(data);
+      var json = haxe.Json.parse(s);
+      return toExpr(json);
```

Comment:
Is there a reason you go yaml->data->json->data?

Why not just:

```
var data = Yaml.parse(....);
return toExpr( data );
```

I'll merge it anyway, but I am curious :)